core/store/src/db.rs

```diff
@@ -783,8 +790,11 @@ impl RocksDB {
         RocksDBOptions::default().read_only(path)
     }
 
-    pub fn new<P: AsRef<std::path::Path>>(path: P) -> Result<Self, DBError> {
-        RocksDBOptions::default().read_write(path)
+    pub fn new<P: AsRef<std::path::Path>>(
+        path: P,
+        extra_db_memory: usize,
+    ) -> Result<Self, DBError> {
+        RocksDBOptions::default().read_write(path, extra_db_memory)
     }
```

Comment:
No need to change this -- if the user wants to create RocskDB with non-default options, they should use `RocksDBOptions`